src/codegen/ast_interpreter.cpp

```diff
@@ -285,32 +285,35 @@ ASTInterpreter::ASTInterpreter(CLFunction* clfunc)
     assert(scope_info);
 }
 
-void ASTInterpreter::initArguments(int nargs, BoxedClosure* _closure, BoxedGenerator* _generator, Box* arg1, Box* arg2,
-                                   Box* arg3, Box** args) {
+void ASTInterpreter::initArguments(BoxedClosure* _closure, BoxedGenerator* _generator, Box* arg1, Box* arg2, Box* arg3,
+                                   Box** args) {
+    RootedBox arg1_root(arg1);
+    RootedBox arg2_root(arg2);
+    RootedBox arg3_root(arg3);
```

Comment:
Concerning the ```RootedBoxes``` I'm trying to remove them again. It looked like it works without them. And I can't reason why the are needed inside this function (in a debug build).
But the next build will tell more...